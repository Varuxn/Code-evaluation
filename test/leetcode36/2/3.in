15
5 -4 7 -5 -5 -6 -5 5 -6 1 -5 7 -7 0 -6 