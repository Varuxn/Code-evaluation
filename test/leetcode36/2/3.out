-7 -6 
-7 0 
-6 -6 
-6 -6 -6 
-6 -6 -5 
-6 -6 -5 0 
-6 -6 -5 7 
-6 -6 0 
-6 -6 1 
-6 -6 1 7 
-6 -6 7 
-6 -5 
-6 -5 -5 
-6 -5 -5 0 
-6 -5 -5 7 
-6 -5 0 
-6 -5 1 
-6 -5 1 7 
-6 -5 5 
-6 -5 5 7 
-6 -5 7 
-6 0 
-6 1 
-6 1 7 
-6 5 
-6 5 7 
-6 7 
-5 -5 
-5 -5 -5 
-5 -5 -5 -5 
-5 -5 -5 -5 0 
-5 -5 -5 -5 7 
-5 -5 -5 0 
-5 -5 -5 1 
-5 -5 -5 1 7 
-5 -5 -5 5 
-5 -5 -5 5 7 
-5 -5 -5 7 
-5 -5 0 
-5 -5 1 
-5 -5 1 7 
-5 -5 5 
-5 -5 5 7 
-5 -5 7 
-5 0 
-5 1 
-5 1 7 
-5 5 
-5 5 7 
-5 7 
-4 0 
-4 1 
-4 1 7 
-4 5 
-4 5 7 
-4 7 
-4 7 7 
1 7 
5 5 
5 5 7 
5 7 
5 7 7 
7 7 
